### PR TITLE
Syncs disabled and expanded in order to toggle dropdown and updates metal-dropdown to version 3.0.0

### PR DESCRIPTION
**Pull request by pragmaticivan**

### CODE REVIEW

File: src/Select.js [hunk 1]
```diff
@@ -172,6 +172,26 @@ class Select extends Component {
 		this.expanded_ = false;
 	}
 
+	/**
+	 * Syncs disabled closing dropdown when select is changed to disabled
+	 * and updates internal state expanded_ to be false.
+	 * @param {boolean} disabled
+	 */
+	syncDisabled(disabled) {
+		if(disabled) {
```
Comment (eduardolundgren) [inline src/Select.js]: Source formatting here `if (`

File: src/Select.js [hunk 2]
```diff
@@ -172,6 +172,26 @@ class Select extends Component {
...
+		}
+	}
+
+	/**
+	 * Syncs internal state expanded_ and toggle dropdown
+	 * @param {boolean} expanded
+	 */
+	syncExpanded_(expanded) {
+		const dropdown = this.getDropdown();
+		expanded ? dropdown.open() : dropdown.close();
```
Comment (eduardolundgren) [inline src/Select.js]: Avoid ternary for fn calls.

```
if (expanded) {
  this.getDropdown().open();
} else {
  this.getDropdown().close();
}
```

**Comment by pragmaticivan** [comment]
@eduardolundgren fixed, thanks 👍

**Comment by eduardolundgren** [comment]
LGTM

**Comment by fernandosouza** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request merged**